### PR TITLE
i9082: Add ZRAM support and remove egl.cfg (deprecated)

### PR DESCRIPTION
**Pull request by luk1337**
256M of ZRAM should help a bit ;3

### CODE REVIEW

File: BoardConfig.mk [hunk 1]
```diff
@@ -73,7 +73,6 @@ BOARD_USES_SKTEXTBOX := true
 
 # Hardware rendering
 USE_OPENGL_RENDERER := true
-BOARD_EGL_CFG := device/samsung/i9082/egl.cfg
```
Comment (yupi731) [inline BoardConfig.mk]: Ok

**Comment by pawitp** [comment]
Will look at this for the next build.

Did you do some daily usage testing with zRAM? My previous attempt on the i9000 resulted in mixed response from the user.

BTW, the i-Mobile IQ II (Android One with 1 GB RAM) came with 512 MB of zRAM enabled by default. That might be a better value. (Though note that the IQ II has a quad-core CPU.)

**Comment by luk1337** [comment]
Well Motorola devices with msm8226 (4 core) and msm8610 (2 core) are using zRAM by default so I thought that wouldn't be a bad idea to enable it here aswell. I can't test how it works on BCM28155 since I sold my S2 Plus but it shouldn't be bad.

**Comment by pawitp** [comment]
It's a trade-off between RAM and CPU, so I'm not too sure about that "shouldn't". I don't use the device daily, so I can't test it myself either.

For reference, can you tell me what are those Motorola devices?

**Comment by luk1337** [comment]
Moto E 1st gen (msm8610), Moto G 1st/2nd gen (msm8226)

**Comment by pawitp** [comment]
Thanks

File: BoardConfig.mk (hunk 1 above)
Comment (yupi731) [inline BoardConfig.mk]: I13536e10c0b67bf9830af91f7ed3c4f3653d11cb

Comment (yupi731) [inline BoardConfig.mk]: 256M of ZRAM should help a bit ;3

Comment (yupi731) [inline BoardConfig.mk]: 256M of ZRAM should help a bit ;3

**Comment by pawitp** [comment]
@luk1337 Ever had any experience with http://forum.xda-developers.com/galaxy-grand-duos/development/rom-cm-12-1-galaxy-grand-duos-i9082-t3073108/post62449548?

Not sure if it's related or not.

**Comment by luk1337** [comment]
I had those glitches when I was using justarchi's optimizations.

**Comment by pawitp** [comment]
Users reported that disabling zRAM fixes the issue, so I will be reverting this.

**Pull request merged**